auth-jwt-service/pom.xml

```diff
@@ -83,9 +83,17 @@
             <Bundle-Description>AuthJWTService component for project</Bundle-Description>
             <Bundle-SymbolicName>io.stargate.auth.jwt</Bundle-SymbolicName>
             <Bundle-Activator>io.stargate.auth.jwt.AuthJWTServiceActivator</Bundle-Activator>
-            <Import-Package>io.stargate.auth, org.osgi.framework</Import-Package>
+            <Import-Package><![CDATA[
+              org.osgi.framework,
+              io.stargate.core.*,
+              io.stargate.auth,
+              io.stargate.db.*,
+              org.apache.cassandra.stargate,
+              org.apache.cassandra.stargate.*,
+              org.javatuples,
+              com.datastax.oss.driver.shaded.guava.*,
+            ]]></Import-Package>
             <Export-Package>!*</Export-Package>
```

Comment:
should the export for all auth packages be as for the `authnz`?

`<Export-Package>io.stargate.auth,io.stargate.auth.entity</Export-Package>`

What this means, export all right?

Reply:
OK export nothing is even better,